src/Phoenix.vue

```diff
@@ -1,27 +1,19 @@
 <template>
-  <div
-    id="Phoenix"
-    :class="['uk-flex uk-flex-column uk-height-1-1', { loginGradient: showGradientBackground}]"
-    :style="showImageBackground ? { 'background-image': 'url('+configuration.theme.logo.background+')',
-      'background-size':'cover'} : {}"
-  >
+  <div id="Phoenix" class="uk-height-1-1">
     <template v-if="!showHeader">
       <router-view name="fullscreen"></router-view>
     </template>
     <template v-else>
       <notification-bar />
       <top-bar></top-bar>
       <side-menu></side-menu>
-      <router-view v-if="this.$route.matched[0].components.appTopbar" class="appTopbar" name="appTopbar"></router-view>
-      <div class="uk-height-1-1 uk-overflow-auto">
-        <router-view name="appContent"></router-view>
-      </div>
+      <router-view name="app" class="$app-container uk-height-1-1"></router-view>
```

Comment:
If an app doesn't have defined scroll, the whole page is then again scrollable. How do we prevent this when a developer creates an app without forcing him to always define the scroll inside of the app?

Reply:
Fixed it.

Reply:
Thanks